src/submodules/io/jld2_integration.jl

```diff
@@ -0,0 +1,199 @@
+using Random
+using JLD2
+
+"""
+    AgentsIO.to_serializable(t)
+
+Return the serializable form of the passed value. This defaults to the value itself,
+unless a more specific method is defined. Define a method for this function and for
```

Comment:
This section doesn't really tell me _when_ I need to care about it. What is a property that is not handled by the Agents parser? Will there be an error if my model is not serializable?